Common/include/geometry/meshreader/CSU2BinaryMeshReaderBase.hpp

```diff
@@ -0,0 +1,114 @@
+/*!
+ * \file CSU2BinaryMeshReaderBase.hpp
+ * \brief Header file for the class CSU2BinaryMeshReaderBase.
+ *        The implementations are in the <i>CSU2BinaryMeshReaderBase.cpp</i> file.
+ * \author T. Economon, E. van der Weide
+ * \version 8.2.0 "Harrier"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2025, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "CSU2MeshReaderBase.hpp"
+#include "../../../include/toolboxes/SwapBytes.hpp"
+
+/*!
+ * \class CSU2BinaryMeshReaderBase
+ * \brief Base class for the reading of a native SU2 binary grid.
+ * \author T. Economon, E. van der Weide
+ */
+class CSU2BinaryMeshReaderBase : public CSU2MeshReaderBase {
+ protected:
+  constexpr static int SU2_STRING_SIZE = 33; /*!< \brief Size of the strings in the SU2 binary mesh file. */
```

Comment:
Is there a reason behind the small-ish limit? Can it be relaxed? I remember some of the CRM meshes having quite long names padded with spaces.